DataDog/dd-trace-cpp
# Add support for APM Remote Configuration

## Description

Remote Configuration allows to remotely configure the library. For now, it is only possible to update the trace sampling rate remotely.

## How to test

An API Key with Remote Configuration enabled is required.

1. Run the `hasher` example.
2. Generate some traces so the service can appear in the Service Catalogue.
3. In Datadog's UI go to the Service Catalogue and select the `dd-trace-cpp-example` service.
4. An APM Instrumentation banner should be visible.

![output_trimmed_enhanced](https://github.com/DataDog/dd-trace-cpp/assets/204994/a0e4aba4-c38f-4354-88ba-af1a812a9fe5)
